src/components/Accordion/Accordion.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <button
+    id="accordion"
+    :class="expanded ? null : 'border-b border-gray-100'"
+    class="cursor-pointer flex justify-between min-w-full mb-6 focus:outline-none focus:ring-1 focus:ring-primary-100 focus:border-transparent"
+    :aria-expanded="expanded"
+    aria-controls="slotContent"
+    role="button"
+    @click="expanded = !expanded"
+  >
+    <h2 class="text-primary-300 font-thin text-lg">
+      <b>{{ title }}</b>
+    </h2>
+    <div
+      class="flex-end"
```

Comment:
small detail: the icon doesn't look vertically centered. Maybe add some top margin?
![image](https://user-images.githubusercontent.com/83967528/153406848-14c39b9e-0ae3-4b9c-bb47-539062f1c961.png)